### DISCUSSION TITLE
A Question Regarding Student Developer Pack

### DISCUSSION THREAD

**Discussion by heysagnik** [community/community#40068] [GitHub Education] [answered]
I have completed 1 year of usage of STUDENT Developer Benefits. Now can I reapply to get the benefits again? As because I am still a student. 
Please let me know.

And If I used GitHub Student Developer Pack benefits to buy a domain name, then can it be renewed for free ?

**Comment by SilasRodrigues19** [comment, marked as answer]
@heysagnik As long as this PRO badge is showing on your profile, it means you still have the Student Developer Pack. Regarding the domain, it is active for free for 1 year. After that you need to buy
